src/main/java/org/apache/commons/compress/compressors/xz/XZUtils.java

```diff
@@ -163,7 +163,7 @@ public static String getCompressedFilename(final String fileName) {
      * @param doCache whether to cache the result
      * @since 1.9
      */
-    public static void setCacheXZAvailablity(final boolean doCache) {
+    public static void setCacheXZAvailability(final boolean doCache) {
```

Comment:
same as above, this would break backwards compatibility.

Reply:
> same as above, this would break backwards compatibility.

done.